### DISCUSSION TITLE
How to Change you username on github ?

### DISCUSSION THREAD

**Discussion by m-hassanqureshi** [community/community#65349] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I have created my github account but now I want to change my username but there is no option and change username button is not clickable as seen below
![image](https://github.com/community/community/assets/140505321/ebbe29a2-d795-4ea3-adfe-e560bf6a7fbc)

**Comment by Piero24** [comment]
@m-hassanqureshi 
Usually occurs when your username and email address are not configured correctly.
It may also be possible that you used an untrustworthy email address when registering.
It means your account is restricted. In this case the only way is to contact GitHub support,
They will tell you how to proceed (probably if the problem is correlated to the mail address you have to add a new mail address and remove that one).
